api/utils/email_util.py

```diff
@@ -0,0 +1,379 @@
+    )
+    log.info("Sending 'Welcome to MAAP (Active User)' email for {} to {}".format(
+        member.get_display_name(),
+        member.email
+    ))
+    email_message.send()
+    log.info("Email Sent!")
+
+
+def send_welcome_to_maap_suspended_user_email(member: Member, base_url):
+
+    environment_info = get_environment_info(base_url)
+    email_message_subj = "Welcome to the Multi-Mission Algorithm and Analysis Platform (MAAP)!"
+
+    # Build HTML body
+    email_message_html = ""
+    template_file = os.getcwd() + "/api/utils/email_templates/welcome_to_maap_suspended_user_alert.html"
+    with open(template_file, "r", encoding=CHARSET) as file:
+        email_message_html = file.read().format(
+            environment_info=environment_info,
+            member_first_name=member.first_name,
+            support_email=settings.EMAIL_SUPPORT,
+            title=email_message_subj,
+        )
+
+    # Build Text body
+    email_message_txt = ""
+    template_file = os.getcwd() + "/api/utils/email_templates/welcome_to_maap_suspended_user_alert.txt"
+    with open(template_file, "r", encoding=CHARSET) as file:
+        email_message_txt = file.read().format(
+            environment_info=environment_info,
+            member_first_name=member.first_name,
+            support_email=settings.EMAIL_SUPPORT,
+            title=email_message_subj,
+        )
+
+    # Create and send the email
+    email_message = Email(
+        settings.EMAIL_SUPPORT,
+        [member.email],
+        email_message_subj,
+        email_message_html,
+        email_message_txt
+    )
+    log.info("Sending 'Welcome to MAAP (Suspended User)' email for {} to {}".format(
+        member.get_display_name(),
+        member.email
+    ))
+    email_message.send()
+    log.info("Email Sent!")
+
+
+def get_environment_info(base_url):
+
+    env = Environments.DIT
+
+    if "0.0.0.0" in base_url or "127.0.0.1" in base_url or "LOCALHOST" in base_url.upper():
+        env = Environments.DIT
+    elif Environments.DIT.value in base_url.upper():
+        env = Environments.DIT
+    elif Environments.UAT.value in base_url.upper():
+        env = Environments.UAT
+    elif Environments.OPS.value in base_url.upper():
+        env = Environments.OPS
+
+    email_header = {
+        EmailFormats.PLAIN: '',
+        EmailFormats.HTML: '',
+    }
+    if env != Environments.OPS:
+        email_header = {
+            EmailFormats.PLAIN.value: "*** THIS EMAIL WAS SENT FROM THE MAAP {} ENVIRONMENT ***".format(
+                env.value.upper()),
+            EmailFormats.HTML.value: """
+                    <div style="width:100%;padding:10px;margin-bottom:10px;text-align:center;background-color:#FFC107;color:#000000;">
+                        <strong>*** THIS EMAIL WAS SENT FROM THE MAAP {} ENVIRONMENT ***</strong>
+                    </div>
+                """.format(env.value.upper())
+        }
+
+    portal_base_url = "https://{}.maap-project.org".format(env.value.lower())
```

Comment:
I think this will work fine for the time being, though at some point we should look at adding a domain name config setting (and assume the presence of the ade, api, and repo subdomains), so that other projects can utilize MAAP using their own domain names.